Resources/Prototypes/Starshine/Entities/Objects/Weapons/Throwable/grenades.yml

```
@@ -0,0 +1,48 @@
+#Grenades
+- type: entity
+  name: bee grenade
+  description: Grenade full of bees... OH NO, NOT THE BEES!
+  parent: [ProjectileGrenadeBase, BaseSyndicateContraband, GrenadeBase]
+  id: UplinkBeeGrenade
```

Саму гранату можно было не переименовывать, а оставить BeeGrenade. "Uplink" добавляется только прототипам товара в аплинке.
```suggestion
  id: BeeGrenade
```